src/types/auth.ts

```diff
@@ -6,6 +6,7 @@ import { Session } from 'next-auth'
 type SessionData = Omit<Session, 'user'> & {
 	accessToken?: string
 	id?: string
+	error?: string
```

Comment:
🙋🏻 Should this reference the new `AuthErrors` enum instead of `string`?

Reply:
Yes! Good catch